discord/src/main/groovy/com/asm/tavern/discord/discord/audio/SongsCommandHandler.groovy

```diff
@@ -60,7 +51,12 @@ class SongsCommandHandler implements CommandHandler {
 					StringBuilder builder = new StringBuilder()
 					songs.forEach({ song ->
 						currentCategory = song.category
-						builder.append("[${song.id}](${DiscordUtils.escapeUrl(song.uri.toString())})\n")
+						if(song.uri.getScheme() == "http" || song.uri.getScheme() == "https") {
```

Comment:
Can't imagine we would ever get a valid 'http' song url

Reply:
This won't catch urls without a scheme either. Could use a try catch and just send all uri's through and just ignore failed parts of the embed. Another thing is we could make sure local file songs are only added with the file scheme and check for getScheme() == "file" though this would mean we have to remove the scheme when passing it through to lavaplayer.